src/staff_augmentation/components/TechSolutions/styles.module.scss

```diff
@@ -0,0 +1,140 @@
+@import "/src/styles/theme.scss";
+@import "/src/styles/media_queries.scss";
+
+.subSection {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  & > .title {
+    margin-bottom: 50px;
+  }
+
+  & + & {
+    margin-top: 50px;
+
+    @include for-tablet-up {
+      margin-top: 140px;
+    }
+  }
+
+  @include for-tablet-up {
+    flex-direction: row;
+
+    & > .title {
+      display: none;
+    }
+  }
+}
+
+.leftImageContainer,
+.rightImageContainer {
+  position: relative;
+  width: 100%;
+  max-width: 530px;
+  padding-bottom: 26px;
+  flex-grow: 0;
+  height: fit-content;
+
+  @include for-tablet-up {
+    min-width: calc(50% - 20px);
+  }
+
+  > img {
+    width: 100%;
+    height: auto;
+    border-radius: 0px 30px 0px 40px;
+  }
+
+  &::before {
+    content: '';
+    border-radius: 0px 30px 0px 40px;
+    width: calc(100% - 26px);
+    height: calc(100% - 26px);
+    z-index: -1;
+    position: absolute;
+    bottom: 0;
+  }
+}
+
+.leftImageContainer {
+  padding-left: 26px;
+  margin: 0 auto 20px;
+
+  @include for-tablet-up {
+    margin: auto 20px auto 0;
+  }
+
+  &::before {
+    background: rgba($color:#773E9A, $alpha: 0.85);
+    left: 0;
+  }
+}
+
+.rightImageContainer {
+  padding-right: 26px;
+  margin: 0 auto 20px;
+
+  @include for-tablet-up {
+    margin: auto 0 auto 20px;
+    order: 3;
+  }
+
+  &::before {
+    background: rgba($color:#F8B75F, $alpha: 0.85);
+    right: 0;
+  }
+}
+
+.title {
+  letter-spacing: 1.2px;
+  padding-left: 16px;
+  border-left: 9px solid #773E9A;
+}
+
+.textContainer {
+  & > .title {
+    display: none;
+    padding-left: 25px;
+    line-height: 60px;
+
+    @include for-tablet-up {
+      display: block;
+    }
+  }
+
+  @include for-tablet-up {
+    & > p {
+      padding-left: 25px;
+    }
+  }
+}
+
+.upgradeButton {
+  margin-left: 25px;
+}
+
+.hiddenTextContainer {
+  @include for-phone-only {
+    display: none;
+  }
+}
+
+.circleEmpty {
+  height: 77%;
+  width: 32%;
+  border: 6px solid #F8B75F;
+  border-radius: 50%;
+  position: absolute;
+  top: -65%;
+  right: -316px;
+  overflow: hidden;
```

Comment:
In this case, setting the Overflow property here doesn't work, so a horizontal scroll appears because the Circle still goes outside the section width.

The correct place to set the Overflow property is in the parent element (.techSolutionsSection) of the circle shape, exactly on line 7 of the DeliverTechSolutions stylesheet.